src/middlewares/authValidation.js

```diff
@@ -13,11 +9,23 @@ const { errorResponse } = responseHandlers;
 const signupValidation = async (req, res, next) => {
     const { error } = validateSignup(req.body);
     if (error) {
-        const { details } = error;
-    const messages = details.map((err) => err.message.replace(/['"]/g, '')).join(', ');
-    return errorResponse(res, statusCodes.badRequest, messages);
+      displayErrorMessages(error, res);
   }
     return next();
 };
 
-export default { signupValidation };
+/**
+ * @param {object} req
+ * @param {object} res
+ * @param {object} next
+ * @returns{object} return next() if validations pass
+ */
+const passwordResetValidation = async (req, res, next) => {

```

Comment:
I think there is no point of creating the second validation function, you should use the existing validation function which was created to validate signup, and then you edit that function to make it adapt to your needs